twstock/codes.py

```diff
@@ -29,11 +29,12 @@ def read_csv(path, types):
         csvfile.readline()
         for row in reader:
             row = ROW(*row)
```

Comment:
Can we strip the code at this line like `ROW(*row.strip())` so that we don't need to deal with other parts?

Reply:
You mean strip all fields?
I will do that later.